merlin/analysis/globalalign.py

```diff
@@ -156,6 +169,12 @@ def get_global_extent(self):
 
         return minX, minY, maxX, maxY
 
+    def get_fov_boxes(self) -> List:
+        fovs = self.dataSet.get_fovs()
```

Comment:
Can the implementation be moved up to GlobalAlignment? It seems everything that is required is there and this should be the same for all GlobalAlignment subclasses.

Reply:
Sure, done